src/lib/mongoBackend/MongoCommonUpdate.cpp

```diff
@@ -1272,13 +1279,16 @@ static bool addTriggeredSubscriptions_noCache
       long long           lastNotification   = sub.hasField(CSUB_LASTNOTIFICATION)? getIntOrLongFieldAsLongF(sub, CSUB_LASTNOTIFICATION) : -1;
       std::string         renderFormatString = sub.hasField(CSUB_FORMAT)? getStringFieldF(sub, CSUB_FORMAT) : "";
       RenderFormat        renderFormat       = stringToRenderFormat(renderFormatString);
+      ngsiv2::HttpInfo    httpInfo;
+
+      httpInfo.url = sub.hasField(CSUB_REFERENCE)? getStringFieldF(sub, CSUB_REFERENCE) : "";
 
```

Comment:
What about "FIXME PR-where-PR-means-not-this-PR-but-the-master-feature-to-develop-PR-that-will-include this one" ?

xDDD